dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details_facilitator.html.haml

```diff
@@ -0,0 +1,82 @@
+- pre_work_url = 'https://docs.google.com/document/d/1mmBcMoRKO2TVxQSEEb-SxMnhuDZUkjE06tHa-8CGeyE/edit'
+- zoom_download_url = 'https://zoom.us/download'
+- learning_tips_url = 'https://docs.google.com/document/d/e/2PACX-1vQbiURoyzHZhWrmT8jbXq7QRiCMicUIO58EQQcw7aIPdZrxhfn1XP64djtg2TVFTsBNEUZnSYHSx4Lx/pub'
+- digital_digest_url = 'https://bit.ly/summer-prep-sessions'
+
+= render partial: 'workshop_logistics'
+
+= render partial: 'workshop_notes'
+
+%h3 Materials
+
+%p
+  Please have the following with you on the day of your session to ensure you
+  are able to participate fully.
+
+%ul
+  %li
+    Laptop and charger, as well as any adapters. Review
+    = link_to 'this page', Pd::WorkshopMailer::SUPPORTED_TECH_URL
+    for more information regarding compatible operating systems and browsers.
+    We do not recommend bringing a tablet as your primary device.
+  %li Download the Zoom app (more information below)
```

Comment:
`.` at end to be consistent with other bullets?

Reply:
Done.